src/server/db/models/github-user.js

```diff
@@ -9,7 +9,7 @@ import db from '../index';
  *   updated_at: update date
  *   last_login_at: last login date
  */
-export const GitHubUser = db.Model.extend({
+export const GitHubUser = db.model('GitHubUser', {
```

Comment:
Does it need to be exported? It seems like we don't import it directly anymore (as it's always used via `db`.